### ISSUE TITLE
Show PBs/NRs/WRs

### ISSUE THREAD

**Issue by jfly** [cubing/ccm#20]
This will require importing the latest wca db export. I think we can punt on this for Phase 1.

**Comment by coder13** [comment]
Should this instead query the wca site rather than import the latest wca db export? This would stay the most up to date but require the wca site be prepared.

**Comment by jfly** [comment]
I'd be open to the idea of adding an API to facilitate this to the WCA website, however I'm not sure what that would look like. I feel that if we're not clever, we'll end up querying the WCA website for the same thing many times.

**Comment by FatBoyXPC** [comment]
Since those records don't change much, could we calculate them every time we submit results, then cache them somewhere?

**Comment by jfly** [comment]
Caching them "somewhere" sounds like it might end up being the same amount of work as just slurping in the WCA db export. One nice thing about the WCA db export is we never need to worry about cache invalidation, we just clobber everything.

**Comment by FatBoyXPC** [comment]
Yeah, I can definitely see the nice thing about that. However, I really think this is more of an API task. While I'm not convinced we need to stop doing the export (because it could be very useful still), I like having an API point for records I suppose.

**Comment by jfly** [comment]
I could see a web timer being interested in letting you know how you're doing compared to your WCA pbs, so I'm not at all opposed to this.
